client/src/redux/sagas/fixtures/notificationsJobsEvents.fixture.js

```diff
@@ -0,0 +1,205 @@
+    archive: true,
+    archiveOnAppError: true,
+    archivePath:
+      'archive/jobs/2020-09-20/rstudio-stampede2-1-1-423u3_2020-09-20t12-25-10-dcvserver-ca48197a-4e9c-4ffc-868c-29a3113cd5e7-007',
+    archiveSystem: 'frontera.home.maxmustermann',
+    nodeCount: 1,
+    processorsPerNode: 20,
+    memoryPerNode: 1,
+    maxRunTime: '00:06:00',
+    localId: '6477330',
+    batchQueue: 'normal',
+    submitTime: '2020-09-20T12:25:46.170Z',
+    startTime: '2020-09-20T12:25:50.950Z',
+    remoteEnded: null,
+    remoteOutcome: null,
+    submitRetries: 0,
+    remoteStatusChecks: 1,
+    failedStatusChecks: 0,
+    lastStatusCheck: '2020-09-20T12:25:50.945Z',
+    blockedCount: 0,
+    visible: true
+  },
+  pk: 34,
+  action_link: '',
+  user: 'maxmustermann',
+  read: false,
+  deleted: false
+};
+
+export const jobInteractiveSessionReady = {
+  event_type: 'interactive_session_ready',
+  datetime: '1600604773',
+  status: 'INFO',
+  operation: 'web_link',
+  message: 'Ready to view.',
+  extra: {
+    id: 'ca48197a-4e9c-4ffc-868c-29a3113cd5e7-007',
+    name: 'RStudio-Stampede2-1.1.423u3_2020-09-20T12:25:10-dcvserver',
+    tenantId: 'portals',
+    tenantQueue: 'aloe.jobq.portals.submit.DefaultQueue',
+    status: 'RUNNING',
+    lastStatusMessage:
+      'Transitioning from status QUEUED to RUNNING in phase MONITORING.',
+    accepted: '2020-09-20T12:25:28.463-05:00',
+    created: '2020-09-20T12:25:28.000-05:00',
+    ended: null,
+    lastUpdated: '2020-09-20T12:25:50.000-05:00',
+    owner: 'maxmustermann',
+    roles:
+      'Internal/PORTALS_maxmustermann_cep-dev_PRODUCTION,Internal/PORTALS_maxmustermann_test_PRODUCTION,Internal/PORTALS_maxmustermann_cep dev_PRODUCTION,Internal/PORTALS_maxmustermann_test-nathaf_PRODUCTION,Internal/PORTALS_maxmustermann_DefaultApplication_PRODUCTION,Internal/PORTALS_maxmustermann_cep-dev_SANDBOX,Internal/everyone,Internal/PORTALS_maxmustermann_cep dev_SANDBOX',
+    systemId: 'maxmustermann.TACC-ACI.exec.stampede2.HPC',
+    appId: 'prtl.clone.maxmustermann.TACC-ACI.RStudio-Stampede2-1.1.423u3-3.0',
+    appUuid: '6262962608719195670-242ac116-0001-005',
+    workPath:
+      '/scratch/05724/maxmustermann/maxmustermann/job-ca48197a-4e9c-4ffc-868c-29a3113cd5e7-007-rstudio-stampede2-1-1-423u3_2020-09-20t12-25-10-dcvserver',
+    archive: true,
+    archivePath:
+      'archive/jobs/2020-09-20/rstudio-stampede2-1-1-423u3_2020-09-20t12-25-10-dcvserver-ca48197a-4e9c-4ffc-868c-29a3113cd5e7-007',
+    archiveSystem: 'frontera.home.maxmustermann',
+    nodeCount: 1,
+    processorsPerNode: 20,
+    memoryPerNode: 1,
+    maxHours: 0.1,
+    inputs: { workingDirectory: 'agave://frontera.home.maxmustermann/5MB.txt' },
+    parameters: {
+      desktop_resolution: '1280x800',
+      _webhook_base_url: 'http://320a61b9b96e.ngrok.io/webhooks/'
+    },
+    remoteJobId: '6477330',
+    schedulerJobId: null,
+    remoteQueue: 'normal',
+    remoteSubmitted: '2020-09-20T12:25:46.170-05:00',
```

Comment:
These timestamps will change with a fix to the Aloe system. I'll report back - i have a meeting with CIC. It will likely be -00:00, with the timestamp itself not changing since that was the "correct" UTC time, and they are removing the incorrect offset.

Reply:
gotcha.  so we can update these to 0.0.

Reply:
make them 2020-09-20T12:25:46.170Z

Reply:
done